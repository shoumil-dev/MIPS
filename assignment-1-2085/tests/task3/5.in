5
10
3
2
4
10
11
