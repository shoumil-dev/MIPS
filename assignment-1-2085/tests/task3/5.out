MARS 4.5  Copyright 2003-2014 Pete Sanderson and Kenneth Vollmar

Array length: Enter the value of hulk power: Enter num: Enter num: Enter num: Enter num: Enter num: Hulk SMASH! >:(
Hulk SMASH! >:(
Hulk SMASH! >:(
Hulk SMASH! >:(
Hulk Sad :(
Hulk smashed 4 people

